fetch.js

```diff
@@ -167,6 +167,11 @@
 
     this._initBody = function(body) {
       this._bodyInit = body
+
+      delete this._bodyText
+      delete this._bodyBlob
+      delete this._bodyFormData
```

Comment:
I believe you are correct. Since we're now reusing old references, it would always be the same type and thus a delete is unnecessary. I'm removing it now and committing...